pkg/common/metrics/arm.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+// Package metrics provides utilities for collecting ARM API metrics
+package metrics
```

Comment:
follow this instruction,
1. create a customized traffic manager client, refer to the metrics part of https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/pkg/azclient/fileshareclient/custom.go
2. create a unit test for the customized traffic manager client, to validate the metrics
3. use the customized traffic manager client, which contains the emitting arm latency metrics